pkg/liqo-controller-manager/virtualNode-controller/virtualnode_controller.go

```diff
@@ -98,7 +105,7 @@ func filterVirtualNodes() predicate.Predicate {
 		DeleteFunc: func(e event.DeleteEvent) bool {
 			// It is necessary to monitor also the deletion of the NamespaceMap.
 			value, ok := (e.Object.GetLabels())[liqoconst.TypeLabel]
-			return (ok && value == liqoconst.TypeNode) || e.Object.GetNamespace() == liqoconst.TechnicalNamespace
+			return (ok && value == liqoconst.TypeNode) || e.Object.GetNamespace() != ""
```

Comment:
Can you please comment on this very last check in order to make it more explicit?